### DISCUSSION TITLE
Get items for Dovetail Festival of Rail 2024

### DISCUSSION THREAD

**Discussion by woctezuma** [chr233/ASFEnhance#211] [杂谈 / General]
- https://store.steampowered.com/developer/dovetailgames/sale/festivalofrail2024
- https://store.steampowered.com/points/shop/app/2807910
- https://www.steamcardexchange.net/index.php?gamepage-appid-2807910
- https://steamdb.info/app/2807910/communityitems/

Stream:
- https://steamcommunity.com/broadcast/watch/76561199177708161

Command:
```
unknown
```

Reference:
- https://store.steampowered.com/news/app/2362300/view/4017842204442766946

**Comment by chr233** [comment]
background 
Azuma 265619
Metrolink 265620
OBB 265621
FlyingScot 265622
Vectron 265623
Train Simulator Classic: AC4400CW 265624
Train Simulator Classic: Class 390  265625
Train Simulator Classic: ICE T 265626

emoji
265627 265628 265629 265630 265631

**Reply by victoronlyarg** [reply]
Does it work with a command?

**Reply by woctezuma** [reply]
Not this time. It is a broadcast-based giveaway, not in the points shop.
